frontend/src/metabase/visualizations/echarts/cartesian/option/index.ts

```diff
@@ -0,0 +1,37 @@
+import type { EChartsOption } from "echarts";
+import type { CartesianChartModel } from "metabase/visualizations/echarts/cartesian/model/types";
+import { buildOptionMultipleSeries } from "metabase/visualizations/echarts/cartesian/option/series";
+import type {
+  ComputedVisualizationSettings,
+  RenderingContext,
+} from "metabase/visualizations/types";
+import { buildAxes } from "metabase/visualizations/echarts/cartesian/option/axis";
+
+export const buildCartesianChartOption = (
+  chartModel: CartesianChartModel,
+  settings: ComputedVisualizationSettings,
+  renderingContext: RenderingContext,
+): EChartsOption => {
+  const series = buildOptionMultipleSeries(
```

Comment:
Let's name this `echartsSeries` to avoid confusion with our own series/seriesModels (already the naming has gotten very confusing).

I also think we can define type aliases (e.g. type EChartsSeries = EChartsOption["series"]) to get in the habit of using them.